### ISSUE TITLE
JavaScript-loaded content is very slow and often fails to load

### ISSUE THREAD

**Issue by genodeftest** [fedora-infra/fedora-packages#228]
There are several elements that load very slow or often fail to load at all:
1. on a package list, below "Overview", section "Recent History"
2. Tab "Builds", "Updates", "Bugs", "Contents", "Changelog", "Sources": most of the content

Those contents take several seconds, sometimes even minutes, and sometimes timeout.

**Comment by pypingou** [comment]
Would you have a specific example?

**Comment by genodeftest** [comment]
No, it affects all packages. The "Bugs" site of the kernel package seems to be a extreme result due to some 600 bugs being displayed.

This might be specific to having a slow internet connection or some browser configuration. Do you know how I could track that down?

Btw: I'm getting a warning in Firefox inspector:
```
getPreventDefault() should not be used any more. Use defaultPrevented instead.  jquery.js:3446:28
```
Which seems to be there due to a pretty outdated jquery.js apps.fp.o is using. This should be fixed in more recent releases: https://github.com/jquery/jquery/search?q=getPreventDefault&type=Issues&utf8=%E2%9C%93

**Comment by genodeftest** [comment]
Sometimes firefox web inspector console gives me this error message:

```
TypeError: jQuery(...).mokshagrid is not a function contents:447:30
<anonym> contents:447
jQuery.Callbacks/fire() jquery.js:1046
jQuery.Callbacks/self.fireWith() jquery.js:1164
.ready() jquery.js:435
DOMContentLoaded() jquery.js:923
.send() jquery.js:8102
.ajax() jquery.js:7580
evalScript() jquery.js:6427
.each() jquery.js:658
.domManip() jquery.js:5984
.append() jquery.js:5761
.html() jquery.js:5878
load_finish() moksha.js:416
moksha.ajax_load/success() moksha.js:587
jQuery.Callbacks/fire() jquery.js:1046
jQuery.Callbacks/self.fireWith() jquery.js:1164
done() jquery.js:7399
.send/callback() jquery.js:8180
```

**Comment by pypingou** [comment]
I used the ``guake`` project after reading this ticket earlier and it all worked fine for me. Could you try as well?
The bugs tab for the kernel I can understand, but Changelog, Updates and Builds are taking a little time but they do show up, Sources as well but it was the slowest the first time I accessed it.

**Comment by genodeftest** [comment]
Still doesn't work. Not a permanent issue though, sometimes it works sometimes it fails. When it fails it tends to fail on every page load for several minutes.

**Comment by techtonik** [comment]
Very slow. Would be nice to get metrics about request into web console.
